docs/source/gs-processing/usage/emr-serverless.rst

```diff
@@ -87,15 +87,15 @@ Here we will just show the custom image application creation using the AWS CLI:
 .. code-block:: bash
 
     aws emr-serverless create-application \
-        --name gsprocessing-0.2.1 \
+        --name gsprocessing-0.2.2 \
```

Comment:
Shall we use latest here as what we do in the SageMaker part?

Reply:
Because the version of EMR we use is tied to the library version, we keep this specific.